framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -366,6 +381,121 @@ public S initialStore(UnderlyingAST underlyingAST, List<LocalVariableNode> param
     return store;
   }
 
+  /**
+   * Determine whether a lambda expression is leaked.
+   *
+   * <p>This currently checks for the {@link NonLeaked} annotation, which is trusted.
```

Comment:
The `NonLeaked` annotation is not related to the lambda.  It is for method calls within the lambda.

Reply:
I think I misunderstood what needs to be unleaked; I was under the impression that the following annotation in the JDK (linked in https://github.com/typetools/jdk/pull/202) was adequate:

```java
default void forEach(@NonLeaked Consumer<? super T> action)
```

Is this no longer the case?

If it is no longer the case, where would a programmer supply the `@NonLeaked` annotation in the code below?


```java
 void test(OptContainer container, List<String> strs) {
     if (container.getOpt().isPresent()) {
       strs.forEach(s -> container.getOpt().get()); // Legal
     }
   }
```

Reply:
I misspoke:  the `NonLeaked` annotation is not on the lambda.  It is on the declaration of a method that the lambda is being passed to.  Please clarify, in the comment, where the annotation is looked for.